src/components/Reservations/Reservation.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Reservation = (props) => {
+  const {
+    userId, roomId, startDate, endDate, city,
+  } = props;
+
+  return (
+    <div className="reservation">
+      <p>
+        user id:
+        {' '}
+        {userId}
```

Comment:
- Dear @AxelSoler, this is great. Also I think this might be improved as `<p>{"user id: "+userId}</p>`

Reply:
Sadly I have linter errors with that solution